shared/pingpong/pingpong.go

```diff
@@ -141,6 +146,12 @@ func computeAccountMinBalance(cfg PpConfig) (requiredBalance uint64) {
 	}
 	if cfg.MaxFee != 0 {
 		fee = cfg.MaxFee
+	} else {
+		// follow the same logic as constructTxn
+		fee, err = client.SuggestedFee()
```

Comment:
This is correct - functionally. However, it will reduce the pingpong throughput..

Reply:
this is called only for funding accounts that is done in between test runs. I guess one call per `--duration N` is not a big deal.

Reply:
ahh. I didn't get that it's only one call per batch. In this case, I think that it's fine.

Reply:
Was just running the `ineffassign` linter and saw that this change makes it so the `if cfg.MinFee > fee { fee = cfg.MinFee }` section above is ineffectual/useless, since this else expression will overwrite whatever fee might have been set above. Is that OK? Should I remove the `if cfg.MinFee > fee` code?